### DISCUSSION TITLE
Strange action in act 2

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#293] [Q&A]
Hi. If my charactor start in act 2 town (died/game closed in act2 last time)
my charactor going anywhere(not to NPC just nowhere) and spend a lots of time in act 2 town.
How can I fix this?

**Comment by isid0re** [comment]
That issue isnt script related and isnt something that can be fixed by changing a setting. Its within the d2bot.exe itself.
